.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1

```

Comment:
This file should not be included in the PR.

I recommend adding `/.settings` to your global git ignore. See https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore for how to do that.